src/how-tos/amazon-msk/secure-public-access/development.md

```diff
@@ -91,7 +91,7 @@ Your IP may be different when you SSH into the EC2 instance. VPNs and other netw
 Navigate to the VPC Management Console [Security Groups](https://console.aws.amazon.com/vpc/home#securityGroups:) table.
 
 ::: note Check your selected region
-Make sure you have selected the desired region, such as `US East (N. Virginia) us-east-1`.
+Make sure you have selected the desired region, ex: `US East (N. Virginia) us-east-1`.
```

Comment:
All of these are because the default page with was wrapping so it needed to be a few characters shorter